src/devices/gmmts.ts

```diff
@@ -0,0 +1,848 @@
+import {Definition, Fz, Tz, KeyValue, Zh} from '../lib/types';
```

Comment:
This is quite some code to support just a single device, would it be possible to simply somehow by relying on standard functionality like reporting?

Reply:
This device is a smart energy monitor, witch is pluged to a electric meter and read data.
Depending of the electricity contract, commincation mode, number of phase, eletricity producer or not, entitys appears or not.
I already use reporting for somes attributes.
I use custom fz for haElectricalMeasurement beacause the device dont have all attributes available.

I dont know how to reduce the code.

Thanks

Reply:
I edited some fzlocal to reduce the code size.